assets/js/vue/components/search/DatasetRow.vue

```diff
@@ -44,7 +44,7 @@
         },
         data: function () {
             return {
-                url: Routing.generate("pelagos_app_ui_dataland_default", {'udi' : this.datasetRowData.udi } )
+                url: this.datasetRowData.uri
```

Comment:
I guess we could have just named it the new search field url, and then skipped this part. Or call URI instead or URL in the window.open.